Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -66,22 +96,27 @@ final class CategoryContentsViewController: BaseViewController {
                                      style: .plain,
                                      target: self,
                                      action: #selector(goToCategoryCorrection(_:)))
+        button.tintColor = .gray003
+        let titleAttributes = [
+            NSAttributedString.Key.font: UIFont.font(FontName.pretendardMedium, ofSize: CGFloat(14))
+            ]
```

Comment:
p1;

```suggestion
        let titleAttributes = [
            .font: UIFont.font(.pretendardMedium, ofSize: 14)
            ]
```
이렇게 써도 될 거 같아요!

Reply:
3e0c780feff586bc41